sdk/keyvault/azure-security-keyvault-common/inc/azure/keyvault/common/internal/keyvault_pipeline.hpp

```diff
@@ -141,7 +141,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Common { n
      * @param path The path for the request.
      * @return A unique ptr to an Http raw response.
      */
-    std::unique_ptr<Azure::Core::Http::RawResponse> GetResponse(
+    std::unique_ptr<Azure::Core::Http::RawResponse> ExtractResponse(
```

Comment:
doesn't make sense. The object is not a container for the Response, if we don't like `GetResponse`, then it should be something like `BuildResponse` or `SendResponse`.
It's like renaming the `Send` method from the pipeline as `ExctractResponse` just because it returns a unique_ptr.

Also, this is an internal method, do we want mark `GetResponse` and `ExtractResponse` as reserved keywords for public API undersatanding?

Reply:
Nothing called `Get()` should be transferring ownership. So, Extract() is not a bad name. others would be `Take()`, `Create()`, or context-dependent things such as `Send()`.